test/functional/plugin/man_spec.lua

```diff
@@ -3,6 +3,14 @@ local Screen = require('test.functional.ui.screen')
 local command, eval, rawfeed = helpers.command, helpers.eval, helpers.rawfeed
 local clear = helpers.clear
 
+-- For some reason these tests fail on big endian systems and windows
+-- systems with missing :endif errors which make no sense.
+-- https://travis-ci.org/github/neovim/neovim/jobs/719200512
+if helpers.pending_win32(pending) then return end
+-- https://stackoverflow.com/a/23596380/4283659
+-- http://lua-users.org/lists/lua-l/2008-11/msg00075.html
+if not string.dump(function() end):byte(7) then return end
```

Comment:
this would be worth putting in helpers.lua assuming we don't already have it somewhere (`git grep string.dump.*byte`)

Reply:
Got it.